For the simulation.py example I'm getting a KeyError for 'default_torsion_force_constant' never being assigned.

`Traceback (most recent call last):
  File "simulation.py", line 110, in <module>
    monomer_types=[A],
  File "/projects/chwa2399/software/anaconda/envs/cg_openmm_py37/lib/python3.7/site-packages/foldamers-1.12-py3.7.egg/foldamers/cg_model/cgmodel.py", line 235, in __init__
    self.system = build_system(self, rosetta_functional_form=rosetta_functional_form)
  File "/projects/chwa2399/software/anaconda/envs/cg_openmm_py37/lib/python3.7/site-packages/cg_openmm-1.0-py3.7.egg/cg_openmm/build/cg_build.py", line 816, in build_system
    cgmodel, torsion_force = add_force(cgmodel, force_type="Torsion")
  File "/projects/chwa2399/software/anaconda/envs/cg_openmm_py37/lib/python3.7/site-packages/cg_openmm-1.0-py3.7.egg/cg_openmm/build/cg_build.py", line 693, in add_force
    torsion_force_constant = cgmodel.get_torsion_force_constant(torsion)
  File "/projects/chwa2399/software/anaconda/envs/cg_openmm_py37/lib/python3.7/site-packages/foldamers-1.12-py3.7.egg/foldamers/cg_model/cgmodel.py", line 1176, in get_torsion_force_constant
    return self._get_bonded_parameter(particle_types, "torsion_force_constants")
  File "/projects/chwa2399/software/anaconda/envs/cg_openmm_py37/lib/python3.7/site-packages/foldamers-1.12-py3.7.egg/foldamers/cg_model/cgmodel.py", line 1006, in _get_bonded_parameter
    default_value = forces[default_name]
KeyError: 'default_torsion_force_constant'`

--- a/cg_openmm/utilities/iotools.py
+++ b/cg_openmm/utilities/iotools.py
@@ -120,12 +120,11 @@ def write_pdbfile_without_topology(CGModel, filename, energy=None):
 
     coordinates = CGModel.positions
     old_monomer_index = 0
-
     for particle in CGModel.particle_list:
         bead_index = CGModel.get_particle_index(particle)
-        particle_name = CGModel.get_particle_name(particle)
-        if len(particle_name) > 3:
-            particle_name = particle_name[0:3]
+        particle_type_name = CGModel.get_particle_type(particle)['particle_type_name']
+        if len(particle_type_name) > 3:
+            particle_type_name = particle_type_name[0:3]
         monomer_name = CGModel.get_particle_monomer_type(particle)["monomer_name"]
         if len(monomer_name) > 3:
             monomer_name = monomer_name[0:3]
@@ -134,7 +133,7 @@ def write_pdbfile_without_topology(CGModel, filename, energy=None):
             old_monomer_index = monomer_index
             element_index = 1
         pdb_object.write(
-            f"ATOM{bead_index+1:>7d} {particle_name:>3s}{element_index} {monomer_name:>3s} A{monomer_index:>4}    "
+            f"ATOM{bead_index+1:>7d} {particle_type_name:>3s}{element_index} {monomer_name:>3s} A{monomer_index:>4}    "
             f"{coordinates[bead_index][0].value_in_unit(unit.angstrom):>8.3f}"
             f"{coordinates[bead_index][1].value_in_unit(unit.angstrom):>8.3f}"
             f"{coordinates[bead_index][2].value_in_unit(unit.angstrom):>8.3f}"

--- a/cg_openmm/utilities/iotools.py
+++ b/cg_openmm/utilities/iotools.py
@@ -123,13 +123,18 @@ def write_pdbfile_without_topology(CGModel, filename, energy=None):
 
     for particle in CGModel.particle_list:
         bead_index = CGModel.get_particle_index(particle)
+        particle_name = CGModel.get_particle_name(particle)
+        if len(particle_name) > 3:
+            particle_name = particle_name[0:3]
         monomer_name = CGModel.get_particle_monomer_type(particle)["monomer_name"]
+        if len(monomer_name) > 3:
+            monomer_name = monomer_name[0:3]
         monomer_index = CGModel.get_particle_monomer(particle)+1
         if old_monomer_index < monomer_index:
-            element_index = 1
             old_monomer_index = monomer_index
+            element_index = 1
         pdb_object.write(
-            f"ATOM{bead_index+1:>7d}  X{element_index}   {monomer_name}  A{monomer_index:>4}    "
+            f"ATOM{bead_index+1:>7d} {particle_name:>3s}{element_index} {monomer_name:>3s} A{monomer_index:>4}    "
             f"{coordinates[bead_index][0].value_in_unit(unit.angstrom):>8.3f}"
             f"{coordinates[bead_index][1].value_in_unit(unit.angstrom):>8.3f}"
             f"{coordinates[bead_index][2].value_in_unit(unit.angstrom):>8.3f}"

